core/capabilities/webapi/outgoing_connector_handler.go

```diff
@@ -11,28 +12,45 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/connector"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/capabilities"
 	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/common"
-	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/webapicapabilities"
 )
 
-var _ connector.GatewayConnectorHandler = &ConnectorHandler{}
+var _ connector.GatewayConnectorHandler = &OutgoingConnectorHandler{}
 
-type ConnectorHandler struct {
+type OutgoingConnectorHandler struct {
```

Comment:
what makes this implementation specifically outgoing and is there an incoming handler?

Reply:
we agreed on this name because this handler would take care of `http target` and `fetch` which would be outgoing, the incoming one would be`http trigger`. this same outgoing/incoming naming is used in the gateway handlers [doc](https://docs.google.com/document/d/1sir1PvAt418CCQlzE41BUqSNMmGme0qA5NlDbl0isX4/edit?tab=t.0#heading=h.g1t6sp1gqdh9)

Reply:
thanks